utils.sh

```diff
@@ -8,6 +8,11 @@ function generate_assets() {
       echo "Templating ${file} to assets/generated/${file}"
       cp assets/{templates,generated}/${file}
 
+      # FIXME: TEMPLATING FOR METAL3 CONFIGMAP
```

Comment:
We need to copy the template before sedding I think - see my branch where I added a function that does that, one thing I'm not yet clear on is how we make that configmap appear in the openshift-machine-api namespace via a manifest, vs the --namespace arg on the CLI (I guess we can specify that context in the manifest, but not familiar with the exact syntax yet)

Reply:
Ah got it, I don't want to duplicate effort since it seems like you're further along.

I think you can add `namespace` to metadata, at least according to https://unofficial-kubernetes.readthedocs.io/en/latest/tasks/configure-pod-container/configmap/:


```yaml
kind: ConfigMap
apiVersion: v1
metadata:
  name: example-config
  namespace: default
```

Reply:
Ack thanks, makes sense - updated my branch and re-testing now